app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,38 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
```

Comment:
Nice work here, but endeavor to align with the project requirement,  which says we should display the recipe of the logged in user.
```suggestion
  def index
    @recipes = Recipe.where(user_id: current_user.id)
   end
```